src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -168,9 +168,16 @@ For example, `sub-01_mod-T1w_defacemask.nii.gz`.
 If several scans of the same modality are acquired they MUST be indexed with the
 [`run-<index>`](../99-appendices/09-entities.md#run) key-value pair:
 `_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
-run labels). When there is only one scan of a given type the run key MAY be
-omitted. Please note that diffusion imaging data is stored elsewhere (see
-below).
+run labels).
+If several scans with the same acquisition parameters are acquired in the same session,
+they MUST be indexed with the [`run-<index>`](../99-appendices/09-entities.md#run) entity:
+`_run-1`, `_run-2`, `_run-3`, and so on (only nonnegative integers are allowed as
+run labels).
+
+If different entities apply,
+such as a different session indicated by [`ses-<label>`](../99-appendices/09-entities.md#ses),
+or different acquisition parameters indicated by [`acq-<label>`](../99-appendices/09-entities.md#acq),
+then `run` is not needed to distinguish the scans and MAY be omitted.
```

Comment:
I think this is crucial information that should be reflected in the run entity definition, rather than in a specific modality's section.